.github/actions/aurora-manage-cluster/action.yml

```diff
@@ -0,0 +1,202 @@
+name: Deploy RDS Aurora Cluster
+
+description: |
+  This GitHub Action automates the deployment of an RDS Aurora cluster using Terraform.
+  This action will also install Terraform and awscli. It will output the Aurora cluster endpoint.
+
+inputs:
+  cluster-name:
+    description: 'Name of the RDS Aurora cluster to deploy'
+    required: true
+
+  engine-version:
+    description: 'Version of the Aurora engine to use'
+    required: true
+    # renovate: datasource=endoflife-date depName=amazon-rds-postgresql versioning=semver
+    default: "15.4"
```

Comment:
as mentioned in the readme. Is it really required if you supply a default?

Reply:
same applies to all the other inputs.
Some you actually do require: false (which you don't have to mention) and have a default